File: domjudge-3.1.3/tests/test-compile-warning.c
/* $Id: test-compile-warning.c 3302 2010-08-09 18:39:20Z eldering $
 *
 * This should give compiler warnings and fail with NO-OUTPUT
 *
 * @EXPECTED_RESULTS@: NO-OUTPUT
 */

#include <stdio.h>

int main()
{
	char str[1000];

	gets(str);

	return 0;
}

File: domjudge-3.1.3/tests/test-compile-warning.pas
(* $Id: test-compile-warning.pas 3302 2010-08-09 18:39:20Z eldering $
 *
 * This should give a compiler warning and fail with NO-OUTPUT
 *
 * @EXPECTED_RESULTS@: NO-OUTPUT
 *)

program warning(input, output);

var
   a : integer;

begin
   a := 1;
end.
